app/controllers/support/providers_controller.rb

```diff
@@ -1,7 +1,7 @@
 module Support
   class ProvidersController < ApplicationController
     def index
-      @pagy, @providers = pagy(Provider.order(:provider_name).includes(:courses, :users))
+      @providers = Provider.order(:provider_name).includes(:courses, :users).page(params[:page] || 1)
```

Comment:
```suggestion
      @providers = RecuritmentCycle.current.providers.order(:provider_name).includes(:courses, :users).page(params[:page] || 1)
```

We probably only need the current recruitment cycle, else we may need to display it.